### PR TITLE
Server blacklist and Bittensor version fix

### PR DESCRIPTION
**Pull request by Eugene-hu**
- Correctly returns unauthenticated errors due to blacklisting
- updates bittensor versioning such that higher versions such (4.11.0) are permitted by design.
- Remove legacy signature

### CODE REVIEW

File: bittensor/_neuron/text/core_server/__init__.py [hunk 1]
```diff
@@ -634,7 +637,7 @@ def time_check():
             registration_check()
             time_check()
             stake_check()            
-            return False
+            return False, ''
```
Comment (camfairchild) [inline bittensor/_neuron/text/core_server/__init__.py]:
```suggestion
            return False, None
```

File: bittensor/_neuron/text/core_server/__init__.py [hunk 2]
```diff
@@ -634,7 +637,7 @@ def time_check():
...
         except Exception as e:
             self.prometheus_counters.labels("blacklisted").inc()
-            return True
+            return True, e
```
Comment (camfairchild) [inline bittensor/_neuron/text/core_server/__init__.py]: Also add type hint to function 
```python
-> Tuple[bool, Optional[Exception]]
```

**Review by camfairchild — CHANGES_REQUESTED** [review]
See: https://github.com/opentensor/bittensor/pull/1091/files#r1102231464

File: bittensor/_neuron/text/core_server/__init__.py (hunk 2 above)
Comment (Eugene-hu) [reply bittensor/_neuron/text/core_server/__init__.py]: thanks! Nice catch, let me fix that

**Comment by Eugene-hu** [comment]
closes the pr in favour of #1240

**Pull request closed without merging**